### PR TITLE
fix(storefront): BCTHEME-289 Add aria label with price to Product Grid Item

### PR DESCRIPTION
**Pull request by yurytut1993**
#### What?

This PR adds aria label top Product Grid Item with its price

#### Tickets / Documentation

[Ticket](https://jira.bigcommerce.com/browse/BCTHEME-289)

#### Screenshots (if appropriate)

price aria label with sale
Screenshot 2020-10-19 at 12 45 31

price range
Screenshot 2020-10-19 at 12 49 18

simple price
Screenshot 2020-10-19 at 12 46 44

### CODE REVIEW

File: templates/components/products/card.html [hunk 1]
```diff
@@ -16,7 +16,35 @@
...
+                        {{/if}}
+                    {{else}}
+                        {{#if price.non_sale_price_without_tax}}
+                            {{theme_settings.pdp-non-sale-price-label}}{{price.non_sale_price_without_tax.formatted}},
+                            {{theme_settings.pdp-sale-price-label}}{{price.without_tax.formatted}}
+                        {{else}}
+                            {{price.without_tax.formatted}}
+                        {{/if}}
+                    {{/if}}
+                {{/and}}
```
Comment (BC-tymurbiedukhin) [inline templates/components/products/card.html:44]: In such case we are able to move this block to separate file

**Review by BC-tymurbiedukhin — DISMISSED** [review]

File: templates/components/products/card.html (hunk 1 above)
Comment (yurytut1993) [reply templates/components/products/card.html]: done

File: lang/en.json [hunk 2]
```diff
@@ -754,6 +754,8 @@
         "description": "Description",
         "price_with_tax": "(Inc. {tax_label})",
         "price_without_tax": "(Ex. {tax_label})",
+        "price_range_from": "from",
+        "price_range_to": "to",
```
Comment (BC-tymurbiedukhin) [inline lang/en.json:758]: Could we use existing from, to fields?

Comment (yurytut1993) [reply lang/en.json]: fair
took shop_by_price_range_aria from en.json you created earlier

**Review by bc-alexsaiannyi — APPROVED** [review]

**Review by BC-tymurbiedukhin — APPROVED** [review]

**Pull request merged**